src/miner.cpp

```diff
@@ -218,7 +218,7 @@ CBlockTemplate* CreateNewForkBlock(bool& bFileNotFound, const int nHeight)
 
 
     // Largest block you're willing to create:
-    unsigned int nBlockMaxSize = (unsigned int)(MAX_BLOCK_SIZE) + 4000000;
+    unsigned int nBlockMaxSize = (unsigned int)(MAX_BLOCK_SIZE) + 1000000;
```

Comment:
This will cause an error during Mainnet launch of larger fork blocks

Reply:
No, it shouldn't. All fork blocks will be under 1MB.